lib/saturn-node.js

```diff
@@ -70,7 +70,8 @@ async function start ({
         const apiUrl = apiMatch[1]
 
         childProcess.stdout.off('data', readyHandler)
-        activities.emit('activity', {
+        activities.submit({
+          source: 'saturn-L2-node',
```

Comment:
❗ 
I believe we are using human-friendly names, i.e.`Saturn`, `Zinnia`, `Bacalhau`. IIUC, they are directly rendered in Station Desktop UI. 

https://github.com/filecoin-station/core/blob/92858297ff092fd83d77f34c1e8371a3237a940b/commands/station.js#L37
https://github.com/filecoin-station/core/blob/92858297ff092fd83d77f34c1e8371a3237a940b/commands/station.js#L47
https://github.com/filecoin-station/core/blob/92858297ff092fd83d77f34c1e8371a3237a940b/commands/station.js#L59

Let's preserve this behaviour, please.

Nice to have: avoid repetition, extract the source string into a constant, or maybe introduce a per-module function for reporting activities that will add the right source value to all activities.

Reply:
Ok, I've refactored to the old behavior where `commands/station.js` owned the strings, now we're passing callbacks to the modules :)